lib/digitizer_block_impl.cc

```diff
@@ -973,8 +973,6 @@ namespace gr {
          std::chrono::duration<float> remaining_poll_duration = std::chrono::high_resolution_clock::now() - poll_start;
```

Comment:
```suggestion
         std::chrono::duration<float> elapsed_poll_duration = std::chrono::high_resolution_clock::now() - poll_start;
```
or
```suggestion
         std::chrono::duration<float> remaining_poll_duration = poll duration - (std::chrono::high_resolution_clock::now() - poll_start);
```
and then check if it is positive or just sleep max(0,remaining_poll_duration).

Reply:
I agree that this is a better naming.

Reply:
+1, `elapsed_poll_duration` fits much better. I picked the first option, since I think it keeps the code more readable.